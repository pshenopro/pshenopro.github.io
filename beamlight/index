<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8">
    <title>Video in KAZAN CITY</title>
    <link rel="stylesheet" href="reset.css">
    <link rel="stylesheet" href="style.css">
  </head>
  <body>

    <!-- Шапка -->
    <div id="header">
      <div class="header_left">
        <span>Видео-ролики для бизнеса в Казани</span>
      </div>
      <div class="header_right">
        <span>8 (843) 999-99-99</span>
        <a href="#">Заказавать звонок</a>
      </div>
    </div>
    <br style="clear:both">

    <!-- меню -->
    <div id="head_menu">
      <ul>
        <li><a href="#">Главная</a></li>
        <li><a href="#">О нас</a></li>
        <li><a href="#">Наши клиенты</a></li>
        <li><a href="#">Виды роликов</a></li>
        <li><a href="#">Портфолио</a></li>
        <li><a href="#">Цены</a></li>
        <li><a href="#">Контакты</a></li>
      </ul>
    </div>

    <!-- Первый блок -->
    <div class="box_1">
      <div><img src="BeamLightLogoWhite.png" alt=""></div>
      <div><span>Создание видео-роликов и проведение съемки </span></div>
      <div><span>для решения маркетинговых , имиджевых и презентационных задач</span></div>
      <div><span>Посмотрите короткое видео о нашей команде</span></div>
      <img class="array_1" src="array_1.png" alt="">
      <a href="#">смотреть видео</a>
      <div><span>Листайте вниз, чтобы узнать больше</span></div>
      <img src="mouse_1.png" alt="">
    </div>

    <!-- второй блок -->
    <div class="box_2">
      <div><span>Немного о BeamLight</span></div>
      <div>
        <!-- Левая форма -->
        <div class="box_2_left" style="margin-right:300px;">
          <div class="box_2_left_1">
            <img src="forma_1.png" alt="">
            <span>Чем мы занимаемся</span>
          </div>
          <div class="box_2_left_2">
            <span>Сегодня видеоролик способен решать сотни различных задач! Мы создаем видеорекламу, которая продает лучше любого менеджера. Мы делаем презентационные ролики, которые способны рассказать и показать больше, чем любая многочасовая лайв-презентация. А наши корпоративные и имиджевые видео позволяют эффективно продемонстрировать лучшие стороны любого продукта, бизнеса или организации.</span>
          </div>
        </div>
        <!-- Правый форма -->
        <div class="box_2_left">
          <div class="box_2_left_1">
            <img src="forma_2.png" alt="">
            <span>Наша команда</span>
          </div>
          <div class="box_2_left_2">
            <span>В нашей команде только творческие люди, у которых за плечами большой опыт в создании видеороликов, репортажной съёмки, составления сценариев и качественном видео-монтаже. В своей работе мы используем только современную и профессиональную технику. Вместе мы - команда профессионалов своего дела.</span>
          </div>
        </div>
      </div>
    </div>

    <!-- Третий блок -->
    <div class="box_3">
      <div>
        <span>Нам доверяют</span>
      </div>
      <div>
        <ul class="logo_par">
          <li><img src="logo_1.png" alt=""></li>
          <li><img src="logo_2.png" alt=""></li>
          <li><img src="logo_3.png" alt=""></li>
          <li><img src="logo_4.png" alt=""></li>
          <li><img src="logo_5.png" alt=""></li>
          <li><img src="logo_6.png" alt=""></li>
          <li><img src="logo_7.png" alt=""></li>
        </ul>
      </div>
    </div>

    <!-- четвертый блок -->
    <div class="box_4">
      <div class="box_4_text_1">
        <span>Какие задачи мы решаем</span>
        <div></div>
        <span>Список наших услуг не ограничивается только видами работ, представленными ниже. Большой опыт всех членов команды позволяет нам реализовывать любые ваши идеи. Обучающий фильм, кино трейлер, видео отзыв, рекламное видео для кинотеатра – обратитесь к нам, и мы найдём решение ваших задач.</span>
      </div>
      <div class="uslugi">
        <div>
          <img src="usluga_img_1.png" alt="">
          <h2>Рекламные видеоролики о товарах и услугах</h2>
          <span>Основная задача ролика проста – продать продукт. Для этого нужно убедить зрителя в полезности, эффективности товара/услуги, продемонстрировать его качества.</span>
        </div>
        <div>
          <img src="usluga_img_2.png" alt="">
          <h2>Презентационные или корпоративные фильмы</h2>
          <span>Фильмы о компании нужны в первую очередь для создания образа компании, поддержания её имиджа. Важно выделить особенности вашей компании и рассказать о них интересно.</span>
        </div>
        <div>
          <img src="usluga_img_3.png" alt="">
          <h2>Видео инфографика, анимационные ролики</h2>
          <span>Сайт – инструмент продаж. Если вы задумываетесь о том, как увеличить его эффективность, обратите внимание на видеоролики. Посетители охотнее посмотрят видео, чем прочтут текст.</span>
        </div>
        <div>
          <img src="usluga_img_4.png" alt="">
          <h2>Съемка мероприятий и промо-роликов</h2>
          <span>Коммерческая съемка мероприятий используется для рекламы следующего мероприятия такого же формата. На мероприятие придет больше людей, если они посмотрят, как прошло предыдущее</span>
        </div>
      </div>
      <div class="box_4_text_2">
        <span>Портфолио</span>
        <div></div>
        <span>Посмотрите наши последние работы</span>
      </div>
    </div>

    <!-- Пятый блок -->
    <div class="box_work">
      <!-- <div><img src="work_1.png" alt=""></div>
      <div><img src="work_2.png" alt=""></div>
      <div><img src="work_3.png" alt=""></div>
      <div><img src="work_4.png" alt=""></div>
      <div><img src="work_5.png" alt=""></div>
      <div><img src="work_6.png" alt=""></div>
      <div><img src="work_7.png" alt=""></div>
      <div><img src="work_8.png" alt=""></div> -->
      <div class="box_work_1"><a href="#">Смотреть</a></div>
      <div class="box_work_2"><a href="#">Смотреть</a></div>
      <div class="box_work_3"><a href="#">Смотреть</a></div>
      <div class="box_work_4"><a href="#">Смотреть</a></div>
      <div class="box_work_5"><a href="#">Смотреть</a></div>
      <div class="box_work_6"><a href="#">Смотреть</a></div>
      <div class="box_work_7"><a href="#">Смотреть</a></div>
      <div class="box_work_8"><a href="#">Смотреть</a></div>
    </div>

    <!-- footer -->
    <footer id="footer">
      <div>
        <img src="bl.png" alt="">
        <p>Видео-ролики для бизнеса в Казани</p>
      </div>
      <div>
        <span>КОНТАКТЫ</span>
        <a href="#"><img src="vk.png" alt=""></a>
        <a href="#"><img src="you.png" alt=""></a>
        <a href="#"><img src="tweet.png" alt=""></a>
        <span>Политика конфиденциальности</span>
      </div>
      <div>
        <span>8 (843) 999-99-99</span>
        <p>mail@mail.com</p>
      </div>
    </footer>
  </body>
</html>
