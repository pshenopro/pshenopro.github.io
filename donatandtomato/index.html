<!DOCTYPE html>
<html>
  <head>
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <meta charset="utf-8">
    <meta name="description" content="Pink-Donut">
    <title>Pink-Donut</title>
    <link rel="stylesheet" href="css/reset.css">
    <link rel="stylesheet" href="css/normalize.css">
    <link rel="stylesheet" href="css/style.css">
  </head>
  <body>

    <section class="container_1">
      <header>
        <nav class="menu">
          <ul class="active">
            <li><a href="#">home</a></li>
            <li><a href="#">what i do</a></li>
            <li><a href="#">my works</a></li>
            <li><a href="#">about</a></li>
            <li><a href="#">contact</a></li>
          </ul>
          <a class="toggle-nav" href="#">|||</a>
        </nav>

        <div>
          <span>Tacos cardigan <br> truffaut Quanted!</span>
        </div>

        <div class="glasses_box">
          <img src="img/glasses.png" alt="">
        </div>

        <div class="see-more_box">
          <span>see more</span>
        </div>
      </header>
    </section>

    <section class="container_2">
      <div class="container_2-inner-box">
        <p class="container_2-leftbox">what i do</p>
        <p class="container_2-rightbox">Fingerstache green juice semiotics quinoa, truffaut letterpress wolf photo booth ugh biodiesel mumblecore brunch. Meh austin vinyl banh mi, gluten-free chillwave scenester kombucha street art. Before they sold out lumbersexual chillwave, cold-pressed mumblecore beard pickled letterpress normcore crucifix.</p>
      </div>

      <div class="container_2-inner-box-1">
        <div>
          <img src="img/headphones.png" alt="">
          <h3>ultricies</h3>
          <p>Four dollar toast messenger bag green juice, organic cliche tofu synth small batch locavore austin. Squid mlkshk authe ntic pinterest, pour-over gastropub XOXO seitan sartorial
humblebrag. </p>
        </div>

        <div>
          <img src="img/monitor.png" alt="">
          <h3>porttitor</h3>
          <p>Tofu mixtape selfies, swag brooklyn banh mi stumptown loca vore yr migas tousled chicharrones sriracha cray leggings. Street art tousled kale chips poutine schlitz. Pitchfork distillery next level, polaroid roof party austin.</p>
        </div>

        <div>
          <img src="img/camera.png" alt="">
          <h3>Curabitur</h3>
          <p>Dreamcatcher deep v austin schlitz drinking vinegar, messen ger bag slow-carb. Migas tumblr kombucha you probably hav en't heard of them +1 chicharrones. Tilde wayfarers craft beer stumptown, humblebrag ugh whatever.</p>
        </div>
      </div>
    </section>

    <section class="container_3">
      <div class="container_3-myworks">
        <p>my works</p>
      </div>

      <div class="container_3-works">
        <img src="img/works.png" alt="">
        <img src="img/works_1.png" alt="">
        <img src="img/works_2.png" alt="">
        <img src="img/works_3.png" alt="">
      </div>

      <div class="container_3-loadmore">
        <span>load more</span>
      </div>

    </section>

    <section class="container_4">
      <div class="container_4-inner">
        <div class="container_4-form">
          <form class="" action="index.html" method="post">
            <span>name:</span>
            <input type="text" name="" value="">

            <span>email:</span>
            <input type="text" name="" value="">

            <span>message:</span>
            <textarea name="name" rows="8" cols="80"></textarea>

            <div>
              <input type="submit" name="" value="send">
            </div>
          </form>
        </div>

        <div class="container_4-contact">
          <p class="container_4-contact-p">contact</p>

          <ul>
            <li><img src="img/facebook.png" alt=""></li>
            <li><img src="img/tweet.png" alt=""></li>
            <li><img src="img/google.png" alt=""></li>
            <li><img src="img/pined.png" alt=""></li>
            <li><img src="img/linked.png" alt=""></li>
          </ul>

          <div class="">
            <p><span>Address:</span><br>9114 Bridge Street,<br>Hillsboro, OR 97124</p>

            <p><span>Phone:</span><br>+48 202 - 555 - 0114</p>

            <p><span>Email:</span><br>contact@yoursite.com </p>
          </div>

        </div>
      </div>
    </section>



    <script src="https://cdnjs.cloudflare.com/ajax/libs/prefixfree/1.0.7/prefixfree.min.js"></script>
    <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
    <script type="text/javascript">
      jQuery(document).ready(function() {
        jQuery('.toggle-nav').click(function(e) {
          jQuery(this).toggleClass('active');
          jQuery('.menu ul').toggleClass('active');
      e.preventDefault();
      });
      });
    </script>
  </body>
</html>
