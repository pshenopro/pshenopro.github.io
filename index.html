<!DOCTYPE HTML>
<html>
	<head>
		<title>Добро пожаловать, меня зовут Николай и я веб разработчик | Welcome to my web page, my name is Nikolay and I'm a web developer</title>
		<meta charset="utf-8" />
		<meta name="description" content="Сделать сайт или нарисовать дизайн? веб разработка и верстка макетов у нас">
		<meta name="viewport" content="width=device-width, initial-scale=1" />
		<link rel="stylesheet" href="assets/css/main.css" />
	</head>
	<body id="top">

		<!-- Header -->
			<header id="header">
				<div class="inner">
					<a href="index.html" class="image avatar"><img src="images/me.jpg" alt="" /></a>
					<h1><strong>WELCOME to my web page</strong> <br>my name is Nikolay and <br>I'm a web developer <br>based in Moscow, Russia</h1>
				</div>
			</header>

		<!-- Main -->
			<div id="main">

				<!-- One -->
					<section id="one">
						<header class="major">
							<h2>WEB DEVELOPER</h2>
						</header>
						<p>Я не буду вам ничего обещать и говорить что мы делаем супер классные сайты за короткое время и это супер не дорого, нет нет нет нет и нет! Я очень простой человек и со мной легко договорится. <br>Конечно у меня есть команда, это Дизайнеры, Художники, Программисты и как же без наших копирайтеров, которые отлично создают текст на сайтах. <br>Я начал свою деятельность в 2016 году и ни разу не пожалел об этом, в 2017 году решил создать команду мечты по <strong><a style="font-style: italic; font-size: 0.5em;" href="https://ru.wikipedia.org/wiki/%D0%92%D0%B5%D0%B1-%D1%80%D0%B0%D0%B7%D1%80%D0%B0%D0%B1%D0%BE%D1%82%D0%BA%D0%B0" target="_blank">ВЕБ РАЗРАБОТКЕ</a></strong> и я это сделал! <br> Ниже вы можете посмотреть все наши работы и связаться с нами для дальнейшего диалога. <br><br><span style="font-style: italic;">Хорошего вам дня</span></p>
						<ul class="actions">
							<li><a href="#three" class="button">НАПИСАТЬ НАМ</a></li>
						</ul>
					</section>

				<!-- Two -->
					<section id="two">
						<h2>Our works / Портфолио</h2>
						<div class="row">
							<article class="6u 12u$(xsmall) ">
								<a href="http://www.anya-grek.ru/" target="_blank" class="image fit thumb"><img src="images/works/page_1.jpg" alt="Анна Грек, Фотограф" /></a>
								<h3>Анна Грек, Фотограф | свадебная фотосъемка Москва</h3>
								<p><a href="http://www.anya-grek.ru/" target="_blank">anya-grek.ru</a></p>
							</article>

							<article class="6u 12u$(xsmall) ">
								<a href="http://elendan.ru" target="_blank" class="image fit thumb"><img src="images/works/page_2.jpg" alt="Мобильная верстка интернет-магазина" /></a>
								<h3>Мобильная верстка <br>интернет-магазина elendan</h3>
								<p><a href="http://elendan.ru" target="_blank">elendan.ru</a></p>
							</article>

							<article class="6u 12u$(xsmall) ">
								<a href="http://lextrade.ru/" target="_blank" class="image fit thumb"><img src="images/works/page_3.jpg" alt="Верстка и поддержка сайта" /></a>
								<h3>Верстка и поддержка сайта <br>торгового оборудования</h3>
								<p><a href="http://lextrade.ru/" target="_blank">lextrade.ru</a></p>
							</article>

							<article class="6u 12u$(xsmall) ">
								<a href="https://pshenopro.github.io/sheisone/index.html" target="_blank" class="image fit thumb"><img src="images/works/page_4.jpg" alt="Верстка сайта для фотомодели" /></a>
								<h3>Верстка сайта для <br> фотомодели</h3>
								<p><a href="https://pshenopro.github.io/sheisone/index.html" target="_blank">KIRA</a></p>
							</article>

							<!-- <article class="6u 12u$(xsmall) work-item">
								<a href="images/works/page_2.jpg" class="image fit thumb"><img src="images/works/page_2.jpg" alt="" /></a>
								<h3>Анна Грек, Фотограф | свадебная фотосъемка Москва <br><a href="http://www.anya-grek.ru/">anya-grek.ru</a></h3>
							</article> -->

						</div>
						<ul class="actions">
							<li><a href="#" class="button" target="_blank">ПОРТФОЛИО</a></li>
						</ul>
					</section>

				<!-- Three -->
					<section id="three">
						<h2>Get In Touch</h2>
						<p>Accumsan pellentesque commodo blandit enim arcu non at amet id arcu magna. Accumsan orci faucibus id eu lorem semper nunc nisi lorem vulputate lorem neque lorem ipsum dolor.</p>
						<div class="row">
							<div class="8u 12u$(small)">
								<!-- <form method="post" action="#">
									<div class="row uniform 50%">
										<div class="6u 12u$(xsmall)"><input type="text" name="name" id="name" placeholder="Name" /></div>
										<div class="6u$ 12u$(xsmall)"><input type="email" name="email" id="email" placeholder="Email" /></div>
										<div class="12u$"><textarea name="message" id="message" placeholder="Message" rows="4"></textarea></div>
									</div>
								</form>
								<ul class="actions">
									<li><input type="submit" value="Send Message" /></li>
								</ul> -->
							</div>
							<div class="4u$ 12u$(small)">
								<ul class="labeled-icons">
									<li>
										<h3 class="icon fa-home"><span class="label">Address</span></h3>
										125363 Moscow, Russia
									</li>
									<li>
										<h3 class="icon fa-mobile"><span class="label">Phone</span></h3>
										<a href="tel:+79663183606">+7(966)318-3606</a>
									</li>
									<li>
										<h3 class="icon fa-envelope-o"><span class="label">Email</span></h3>
										<a href="mailto:n.psheno@gmail.com">n.psheno@gmail.com</a>
									</li>
								</ul>
							</div>
						</div>
					</section>

			</div>

		<!-- Footer -->
			<footer id="footer">
				<div class="inner">
					<ul class="icons">
						<li><a href="https://t.me/pshenopro" class="icon fa-telegram" target="_blank"><span class="label">Telegram</span></a></li>
						<li><a href="https://www.linkedin.com/in/nikolay-pshenichnyy-b6183a33/" class="icon fa-linkedin" target="_blank"><span class="label">Linkedin</span></a></li>
						<li><a href="https://github.com/pshenopro" class="icon fa-github" target="_blank"><span class="label">Github</span></a></li>
						<li><a href="mailto:n.psheno@gmail.com" class="icon fa-envelope-o" target="_blank"><span class="label">Email</span></a></li>
						<li><a href="https://vk.com/aparis" class="icon fa-vk" target="_blank"><span class="label">VK</span></a></li>
					</ul>
				</div>
			</footer>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.poptrox.min.js"></script>
			<script src="assets/js/skel.min.js"></script>
			<script src="assets/js/util.js"></script>
			<!--[if lte IE 8]><script src="assets/js/ie/respond.min.js"></script><![endif]-->
			<script src="assets/js/main.js"></script>
			<script src="https://cdnjs.cloudflare.com/ajax/libs/prefixfree/1.0.7/prefixfree.min.js">

			</script>

	</body>
</html>
