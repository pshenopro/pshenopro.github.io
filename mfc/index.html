<!DOCTYPE html>
<html>
  <head>
    <meta charset="utf-8">
    <title>TEST</title>
    <link rel="stylesheet" href="css/reset.css">
    <link rel="stylesheet" href="css/normalize.css">
    <link rel="stylesheet" href="css/style.css">
    <script src="https://cdnjs.cloudflare.com/ajax/libs/prefixfree/1.0.7/prefixfree.min.js"></script>
  </head>
  <body>

    <!-- хедер -->
    <header id="">
      <div class="header_inner">
        <div class="header_search">
          <input type="text" name="" value="">
        </div>
        <div class="header_menu">
          <div class="header_question-icon">
            <span>?</span>
            <span class="tooltip_question-icon">Help Guide</span>
          </div>
          <div class="header_mail-icon">
            <span id="counter_box">3</span>
          </div>
          <div class="header_language">
            <ul class="select_flag">
              <li><img src="img/uk_flag.png" alt=""><span>EN</span></li>
              <li><img src="img/rus_flag.png" alt=""><span>RU</span></li>
            </ul>
            <span id="" class="arrow_flag"><img src="img/arrow_flag.png" alt=""></span>
          </div>
          <span class="header_logout">log out</span>
        </div>
      </div>
    </header>

    <!-- профайл -->
    <section id="">
      <div class="container">
        <div class="container_profile">
          <div class="profile">
            <img src="img/profile.png" alt="">
            <span>John Smith</span>
            <span class="profile_logout"></span>
          </div>
          <div class="notification">
            <ul>
              <li><span class="">Персональные данные</span><span></span></li>
              <li><span class="active">Уведомления</span><span></span></li>
            </ul>
          </div>
          <div class="profile_link">
            <a class="invite">invite</a>
            <a class="contact">contact us</a>
          </div>
        </div>

        <!-- главный контент -->
        <div class="container_content">
          <table>
            <tr>
              <td class="checkbox"><input type="checkbox" id="check"><label for="check"></label></td>
              <td></td>
              <td class="new_message">Уведомления <span id="mail_counter">(37)</span></td>
              <td></td>
            </tr>
            <tr class="new_message">
              <td class="checkbox"><input type="checkbox" id="check-1"><label for="check-1"></label></td>
              <td>Change password</td>
              <td>Lorem ipsum dolor sit amet, consectetur adipiscing elit</td>
              <td class="date">2018-11-17 15:48:31</td>
            </tr>
            <tr class="new_message">
              <td class="checkbox"><input type="checkbox" id="check-2"><label for="check-2"></label></td>
              <td>Account linked by broker</td>
              <td>Lorem ipsum dolor sit amet, consectetur adipiscing elit</td>
              <td class="date">2018-11-17 15:48:31</td>
            </tr>
            <tr>
              <td class="checkbox"><input type="checkbox" id="check-3"><label for="check-3"></label></td>
              <td>News letters</td>
              <td>Lorem ipsum dolor sit amet, consectetur adipiscing elit</td>
              <td class="date">2018-11-17 15:48:31</td>
            </tr>
            <tr>
              <td class="checkbox"><input type="checkbox" id="check-4"><label for="check-4"></label></td>
              <td>New subscriber</td>
              <td>Lorem ipsum dolor sit amet, consectetur adipiscing elit</td>
              <td class="date">2018-11-17 15:48:31</td>
            </tr>
            <tr>
              <td class="checkbox"><input type="checkbox" id="check-5"><label for="check-5"></label></td>
              <td>Account linked by broker</td>
              <td>Lorem ipsum dolor sit amet</td>
              <td class="date">2018-11-17 15:48:31</td>
            </tr>
            <tr>
              <td class="checkbox"><input type="checkbox" id="check-6"><label for="check-6"></label></td>
              <td>Support</td>
              <td>Lorem ipsum dolor sit amet, consectetur adipiscing elit</td>
              <td class="date">2018-11-17 15:48:31</td>
            </tr>
            <tr>
              <td class="checkbox"><input type="checkbox" id="check-7"><label for="check-7"></label></td>
              <td>Change password</td>
              <td>Lorem ipsum dolor sit amet, consectetur adipiscing elit</td>
              <td class="date">2018-11-17 15:48:31</td>
            </tr>
            <tr>
              <td class="checkbox"><input type="checkbox" id="check-8"><label for="check-8"></label></td>
              <td>Account linked by broker</td>
              <td>Lorem ipsum dolor sit amet</td>
              <td class="date">2018-11-17 15:48:31</td>
            </tr>
            <tr class="new_message">
              <td class="checkbox"><input type="checkbox" id="check-9"><label for="check-9"></label></td>
              <td>Support</td>
              <td>Lorem ipsum dolor sit amet, consectetur adipiscing elit</td>
              <td class="date">2018-11-17 15:48:31</td>
            </tr>
            <tr>
              <td class="checkbox"><input type="checkbox" id="check-10"><label for="check-10"></label></td>
              <td>News letters</td>
              <td>Lorem ipsum dolor sit amet, consectetur adipiscing elit</td>
              <td class="date">2018-11-17 15:48:31</td>
            </tr>
            <tr class="new_message">
              <td class="checkbox"><input type="checkbox" id="check-11"><label for="check-11"></label></td>
              <td>Change password</td>
              <td>Lorem ipsum dolor sit amet</td>
              <td class="date">2018-11-17 15:48:31</td>
            </tr>
            <tr class="new_message">
              <td class="checkbox"><input type="checkbox" id="check-12"><label for="check-12"></label></td>
              <td>Support</td>
              <td>Lorem ipsum dolor sit amet, consectetur adipiscing elit</td>
              <td class="date">2018-11-17 15:48:31</td>
            </tr>
            <tr class="new_message">
              <td class="checkbox"><input type="checkbox" id="check-13"><label for="check-13"></label></td>
              <td>News letters</td>
              <td>Lorem ipsum dolor sit amet, consectetur adipiscing elit</td>
              <td class="date">2018-11-17 15:48:31</td>
            </tr>
            <tr>
              <td class="checkbox"><input type="checkbox" id="check-14"><label for="check-14"></label></td>
              <td>Support</td>
              <td>Lorem ipsum dolor sit amet</td>
              <td class="date">2018-11-17 15:48:31</td>
            </tr>
            <tr class="new_message">
              <td class="checkbox"><input type="checkbox" id="check-15"><label for="check-15"></label></td>
              <td>Change password</td>
              <td>Lorem ipsum dolor sit amet, consectetur adipiscing elit</td>
              <td class="date">2018-11-17 15:48:31</td>
            </tr>
            <tr>
              <td class="checkbox"><input type="checkbox" id="check-16"><label for="check-16"></label></td>
              <td>News letters</td>
              <td>Lorem ipsum dolor sit amet, consectetur adipiscing elit</td>
              <td class="date">2018-11-17 15:48:31</td>
            </tr>
            <tr class="new_message">
              <td class="checkbox"><input type="checkbox" id="check-17"><label for="check-17"></label></td>
              <td>Support</td>
              <td>Lorem ipsum dolor sit amet, consectetur adipiscing elit</td>
              <td class="date">2018-11-17 15:48:31</td>
            </tr>
            <tr class="new_message">
              <td class="checkbox"><input type="checkbox" id="check-18"><label for="check-18"></label></td>
              <td>New subscriber</td>
              <td>Lorem ipsum dolor sit amet</td>
              <td class="date">2018-11-17 15:48:31</td>
            </tr>
            <tr>
              <td class="checkbox"><input type="checkbox" id="check-19"><label for="check-19"></label></td>
              <td>Change password</td>
              <td>Lorem ipsum dolor sit amet, consectetur adipiscing elit</td>
              <td class="date">2018-11-17 15:48:31</td>
            </tr>
            <tr class="new_message">
              <td class="checkbox"><input type="checkbox" id="check-20"><label for="check-20"></label></td>
              <td>News letters</td>
              <td>Lorem ipsum dolor sit amet, consectetur adipiscing elit</td>
              <td class="date">2018-11-17 15:48:31</td>
            </tr>
            <tr>
              <td class="checkbox"><input type="checkbox" id="check-21"><label for="check-21"></label></td>
              <td>New subscriber</td>
              <td>Lorem ipsum dolor sit amet</td>
              <td class="date">2018-11-17 15:48:31</td>
            </tr>
            <tr class="new_message">
              <td class="checkbox"><input type="checkbox" id="check-22"><label for="check-22"></label></td>
              <td>News letters</td>
              <td>Lorem ipsum dolor sit amet, consectetur adipiscing elit</td>
              <td class="date">2018-11-17 15:48:31</td>
            </tr>
            <tr style="border-bottom: 2px solid #ecf6fe">
              <td class="checkbox"><input type="checkbox" id="check-23"><label for="check-23"></label></td>
              <td>Support</td>
              <td>Lorem ipsum dolor sit amet</td>
              <td class="date">2018-11-17 15:48:31</td>
            </tr>
          </table>

          <div class="pagination">
            <a href="#" id="previous"><img src="img/arrow.png" alt=""></a>
            <a href="#">1</a>
            <a href="#" class="active">2</a>
            <a href="#">3</a>
            <a href="#">4</a>
            <a href="#">5</a>
            <span>. . .</span>
            <a href="#">12</a>
            <a href="#">13</a>
            <a href="#" id="next"><img src="img/arrow.png" alt=""></a>
          </div>
        </div>
        <a href="#" class="footer">Сайт_сайт © 2018</a>
      </div>
    </section>
  </body>
</html>
