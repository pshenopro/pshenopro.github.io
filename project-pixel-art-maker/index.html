<!DOCTYPE html>
<html>
<head>
    <title>Pixel Art Maker!</title>
    <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Monoton">
    <link rel="stylesheet" href="styles.css">
    <meta charset="utf-8">
    <meta http-equiv="x-ua-compatible" content="ie=edge">
    <meta name="viewport" content="width=device-width, initial-scale=1">
</head>
<body>
  <h1>Lab: Pixel Art Maker - by .psheno_pro <img src="pixel.jpg" alt=""></h1>
  <p><span>PURPOSE:</span> Draw yourself in pixels!</p>

  <div style="padding:5px;border:1px solid black;width:500px;margin:0 auto;">
    <h2>CHOOSE Grid Size</h2>

    <form id="sizePicker">
      Grid Height:
      <input type="number" id="inputHeight" name="height" min="1" max="100" value="1">
      Grid Width:
      <input type="number" id="inputWeight" name="width" min="1" max="100" value="1">
      <input type="submit">
    </form>
    <br><br>

    <strong><big>Pick A Color</big></strong>
    <input type="color" id="colorPicker">
    <span>OR</span> <a class="reset" href="#">RESET SHEET</a>
  </div>

  <h2>Picture HERE</h2>
  <table id="pixelCanvas"></table>
  <br>

  <h3>Press the button (print screen) on the keyboard to create a screenshot</h3>
  <a href="https://github.com/pshenopro" target="_blank"><img src="https://assets-cdn.github.com/images/modules/open_graph/github-mark.png" height="20px;" alt="GITHUB pshenopro"></a>


  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
  <script type="text/javascript" src="designs.js">

  </script>
</body>
</html>
