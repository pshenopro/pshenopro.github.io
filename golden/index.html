<!DOCTYPE html>
<html lang="en">
<head>
	<meta charset="UTF-8">
	<title>Golden</title>
	<link rel="stylesheet" href="css/reset.css">
	<link rel="stylesheet" href="css/normalize.css">
	<link rel="stylesheet" href="css/style.css">
</head>
<body>

<!-- первый блок -->
<header>
	<div class="container__header clearfix">
		<img src="img/logo.png" alt="" class="logo">
		<nav>
			<ul>
				<li><a href="#">home</a></li>
				<li><a href="#">services</a></li>
				<li><a href="#">portfolio</a></li>
				<li><a href="#">about</a></li>
				<li><a href="#">contact</a></li>
			</ul>
		</nav>
	</div>

	<div class="container__titles">
		<span>Welcome To Our Studio!</span>
		<h1>It’s nice to Meet you</h1>
	</div>

	<a class="container__header-button" href="#">Tell Me More</a>
</header>


<!-- второй блок -->
<section>
	<div class="container__services">
		<div>
			<p>Services</p>
			<p>Proin iaculis purus consequat sem cure.</p>
		</div>

		<div class="container__services-box1">
			<img src="img/service_1.png" alt="">
			<h2>E-Commerce</h2>
			<span>Proin iaculis purus consequat sem cure
digni ssim. Donec porttitora entum suscipit
aenean rhoncus posuere odio in tincidunt.</span>
		</div>

		<div class="container__services-box2">
			<img src="img/service_2.png" alt="">
			<h2>Responsive Web</h2>
			<span>Proin iaculis purus consequat sem cure
digni ssim. Donec porttitora entum suscipit
aenean rhoncus posuere odio in tincidunt.</span>
		</div>

		<div class="container__services-box3">
			<img src="img/service_3.png" alt="">
			<h2>Web Security</h2>
			<span>Proin iaculis purus consequat sem cure
digni ssim. Donec porttitora entum suscipit
aenean rhoncus posuere odio in tincidunt.</span>
		</div>
	</div>
</section>


<!-- третий блок -->
<section>
	<div class="container_portfolio">
		<h2>Our portfolio</h2>
		<p>Proin iaculis purus consequat sem cure. </p>

		<img src="img/work_1.png" alt="">
		<img src="img/work_2.png" alt="">
		<img src="img/work_3.png" alt="">
	</div>
</section>

<!-- четвертый блок -->
<section>
	<div class="container_about">
		<h2>About Us</h2>
		<span>Proin iaculis purus consequat sem cure. </span>


		<div class="abouts">
			<div class="abouts_text">
				<h2><span>July 2010</span><br>Our Humble Beginnings</h2>
				<p>
					Proin iaculis purus consequat sem cure
					digni ssim. Donec porttitora entum suscipit
					aenean rhoncus posuere odio in tincidunt. Proin
					iaculis purus consequat sem cure digni
					ssim. Donec porttitora entum suscipit.
				</p>
			</div>
			<div class="about_img">
				<img src="img/about_1.png" alt="">
			</div>
		</div>


		<div class="abouts abouts_right">
			<div class="about_img">
				<img src="img/about_2.png" alt="">
			</div>
			<div class="abouts_text">
				<h2><span>January 2011</span> <br>Facing Startup Battles</h2>
				<p>
					Proin iaculis purus consequat sem cure
					digni ssim. Donec porttitora entum suscipit
					aenean rhoncus posuere odio in tincidunt. Proin
					iaculis purus consequat sem cure digni
					ssim. Donec porttitora entum suscipit aenean
					rhoncus posuere odio in tincidunt.
				</p>
			</div>
		</div>


		<div class="abouts">
			<div class="abouts_text">
				<h2><span>December 2012</span> <br>Enter The Dark Days</h2>
				<p>
					Proin iaculis purus consequat sem cure
					digni ssim. Donec porttitora entum suscipit
					aenean rhoncus posuere odio in tincidunt. Proin
					iaculis purus consequat sem cure digni.
				</p>
			</div>
			<div class="about_img">
				<img src="img/about_3.png" alt="">
			</div>
		</div>


		<div class="abouts abouts_right">
			<div class="about_img">
				<img src="img/about_4.png" alt="">
			</div>
			<div class="abouts_text">
				<h2><span>february 2014</span> <br>Our Triumph</h2>
				<p>
					Proin iaculis purus consequat sem cure 
					digni ssim. Donec porttitora entum suscipit
					aenean rhoncus posuere odio in tincidunt. Proin
					iaculis purus consequat sem cure digni
					ssim. Donec porttitora entum suscipit aenean.
				</p>
			</div>
		</div>

	</div>
</section>

</body>
</html>
